tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -235,6 +235,7 @@ def test_query_instrument(test_file_catalog_simulation):
         "instrument": "ultra",
         "data_level": "l2",
         "version": "v001",
+        "descriptor": "science",
```

Comment:
```suggestion
        "descriptor": "sci",
```

Reply:
First I thought same too but she had the descriptor as `science` in above line. I will change above line to match this `sci`.